destroy-cluster.nix

```diff
@@ -0,0 +1,11 @@
+with import <nixpkgs> {};
+writeScriptBin "destroy-cluster.sh" ''
+  #!/usr/bin/env bash
+
+  set -e        # exit on error
+  set -o xtrace # print commands
+
+  IO=$(nix-build -A iohk-ops)/bin/iohk-ops
```

Comment:
this could be improved to just refer to io directly

Reply:
@cleverca22 
Replacing `$IO` with `io` does not work saying `io : command not found`.

Script is being called inside a nix' shell.
(` $(nix-build destroy-cluster.nix)/bin/destroy-cluster.sh`)